Source/Core/Common/Arm64Emitter.cpp

```diff
@@ -329,11 +329,33 @@ void ARM64XEmitter::FlushIcacheSection(u8* start, u8* end)
   // Header file says this is equivalent to: sys_icache_invalidate(start, end - start);
   sys_cache_control(kCacheFunctionPrepareForExecution, start, end - start);
 #else
-#ifdef __clang__
-  __clear_cache(start, end);
-#else
-  __builtin___clear_cache(start, end);
-#endif
+  /* Don't rely on GCC's __clear_cache implementation, as it caches
+   * icache/dcache cache line sizes, that can vary between cores on
+   * big.LITTLE architectures. */
```

Comment:
Can you change the comments to use single line comments instead (//) to match dolphin's style guide.